packages/core-p2p/lib/peer.js

```diff
@@ -300,15 +291,23 @@ module.exports = class Peer {
    * @return {Promise}
    */
   async __broadcastTransactions(transactions) {
-    return this.__post(
-      '/peer/transactions',
-      {
-        transactions,
-      },
-      {
-        headers: this.headers,
-        timeout: 8000,
-      },
-    )
+    let response
+
+    try {
+      response = await this.__post(
```

Comment:
We can just return here, there is no point in assigning it to a variable if we are not going to do anything with it afterwards besides returning it.